railties/lib/rails/generators/rails/plugin/templates/Gemfile.tt

```diff
@@ -13,7 +13,7 @@ gemspec
 # Start debugger with binding.b [https://github.com/ruby/debug]
 # gem "debug", ">= 1.0.0"
 <% end -%>
-<% if RUBY_PLATFORM.match?(/bccwin|cygwin|emx|mingw|mswin|wince|java/) -%>
+<% if RUBY_PLATFORM.match?(/mingw|mswin|java/) -%>
```

Comment:
The deleted RUBY_PLATFORM conditions here don't match either the old or new :platforms values for tzinfo-data below. I think there's no reason to have them; they are also very uncommonly used.